components/telemetry-operator/controller/logpipeline/controller.go

```diff
@@ -172,9 +172,13 @@ func (r *Reconciler) reconcileFluentBit(ctx context.Context, name types.Namespac
 	if err := utils.CreateOrUpdateDaemonSet(ctx, r, daemonSet); err != nil {
 		return fmt.Errorf("failed to reconcile fluent bit daemonset: %w", err)
```

Comment:
If lower case is not necessary for Go code, we should words like `DaemonSet` and `ConfigMap` in camel case, see https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#camel-case